packages/webapp/src/utils/user.tsx

```diff
@@ -30,6 +29,14 @@ export function useSignout() {
         await logoutAPI(); // Destroy server session.
 
         await mutate(() => true, undefined, { revalidate: false }); // clean all cache
-        nav('/signin', { replace: true });
+
+        // swr/infinite doesn't currently support clearing cache keys with the
+        // default mechanism. see https://github.com/vercel/swr/issues/2497
+        for (const key of cache.keys()) {
```

Comment:
🍖 Oddly, the default `mutate` recommended snippet for clearing the cache doesn't actually clear all of the cache. It specifically doesn't clear keys related to `swr/infinite`, so this fallback cleans that up.